### PR TITLE
Package simplescreenrecorder

### PR DESCRIPTION
**Pull request by thisven**
As requested by @diovudau on the [Wishlist](https://github.com/osam-cologne/archlinux-proaudio/issues/3#issuecomment-1554416021), I packaged [SimpleScreenRecorder](https://www.maartenbaert.be/simplescreenrecorder) to be maintained here.

### CODE REVIEW

File: aur/packages [hunk 1]
```diff
@@ -39,6 +39,7 @@ rakarrack-plus
 rezonateur
 rtcqs
 sfzlint
+simplescreenrecorder
```
Comment (SpotlightKid) [inline aur/packages:42]: We can only update packages in the AUR, for which we can ad the `osamc` account as a (co-)maintainer, which is not the case here. This line should be removed.

File: packages/simplescreenrecorder/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+_name=ssr
+pkgname=simplescreenrecorder
+pkgver=0.4.4
+pkgrel=1
+pkgdesc='A feature-rich screen recorder that supports X11 and OpenGL'
+arch=(x86_64 aarch64)
+url='https://www.maartenbaert.be/simplescreenrecorder'
+license=(GPL3)
+groups=(pro-audio)
```
Comment (SpotlightKid) [inline packages/simplescreenrecorder/PKGBUILD:12]: I wouldn't classify ssr as audio software, so I don't think the group fits here and should be removed.

File: packages/simplescreenrecorder/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+depends=(alsa-lib desktop-file-utils ffmpeg glu gtk-update-icon-cache jack
+         libgl libpulse libx11 libxext libxfixes libxi libxinerama qt5-base
+         qt5-x11extras)
+makedepends=(git cmake qt5-tools)
+optdepends=('lib32-simplescreenrecorder: OpenGL recording of 32-bit applications
```
Comment (SpotlightKid) [inline packages/simplescreenrecorder/PKGBUILD:17]: We shouldn't depend on AUR packages, even optionally, in packages in our repo. Users should be able to use our repo without using the AUR, IMHO.

File: packages/simplescreenrecorder/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            ')
+source=("https://github.com/MaartenBaert/$_name/archive/refs/tags/$pkgver.tar.gz"
+         ffmpeg5.patch::https://patch-diff.githubusercontent.com/raw/MaartenBaert/ssr/pull/934.patch)
+sha256sums=('69fccf8df6e7285d4df5033eed8748f4eae13b407488906dcf4810a277d557b8'
+            '4b01938615a34127236a21ee0ffa20bbb179c8bfcc5ecef872fc5a246727d2e1')
+
+prepare() {
+  cd $_name-$pkgver
+  mkdir -p build
+  patch -p1 -i ../ffmpeg5.patch # Fix build with ffmpeg 5
```
Comment (SpotlightKid) [inline packages/simplescreenrecorder/PKGBUILD:27]: Shouldn't this be `"$srcdir"/ffmpeg5.patch`?

File: packages/simplescreenrecorder/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+build() {
+  cd $_name-$pkgver/build
+  cmake -DCMAKE_INSTALL_PREFIX="/usr" -DCMAKE_BUILD_TYPE=Release \
+    -DWITH_QT5=on \
+    -DCMAKE_INSTALL_LIBDIR='lib' ../
```
Comment (SpotlightKid) [inline packages/simplescreenrecorder/PKGBUILD:34]: Minor niggle: format command with one `-D` option per line.

**Review by SpotlightKid — CHANGES_REQUESTED** [review]

File: packages/simplescreenrecorder/PKGBUILD (hunk 3 above)
Comment (thisven) [reply packages/simplescreenrecorder/PKGBUILD:17]: And what if we build that package, too?

Comment (SpotlightKid) [reply packages/simplescreenrecorder/PKGBUILD:17]: As discussed in the Matrix chat, that would require enabling the `multilib` repo in our CI.

Let's wait for the input from @cbix on that.

Comment (cbix) [reply packages/simplescreenrecorder/PKGBUILD:17]: I wouldn't add any packages to our main proaudio repo that depend repos other than core/extra/community. If there is some need for it, we may add a separate `proaudio-multilib` repo. Also, recording OpenGL seems to still be an experimental feature and I'd say it's not worth the effort just to support this super tiny edge case :)

Comment (thisven) [reply packages/simplescreenrecorder/PKGBUILD:17]: I see. I put a comment before the `optdepends` line and provide the link to this conversation for transparency on this decision.

Comment (thisven) [reply packages/simplescreenrecorder/PKGBUILD]: OK, that doesn't seem to work very well in the CI. I just remove the line.

File: packages/simplescreenrecorder/PKGBUILD [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+pkgver=0.4.4
+pkgrel=1
+pkgdesc='A feature-rich screen recorder that supports X11 and OpenGL'
+arch=(x86_64 aarch64)
+url='https://www.maartenbaert.be/simplescreenrecorder'
+license=(GPL3)
+depends=(alsa-lib desktop-file-utils ffmpeg glu gtk-update-icon-cache jack
+         libgl libpulse libx11 libxext libxfixes libxi libxinerama qt5-base
+         qt5-x11extras)
+makedepends=(git cmake qt5-tools)
```
Comment (SpotlightKid) [inline packages/simplescreenrecorder/PKGBUILD:16]: Is `git` still needed to build?

Comment (thisven) [reply packages/simplescreenrecorder/PKGBUILD:16]: I don't think so. Let's see...

**Review by SpotlightKid — APPROVED** [review]
LGTM. Be sure to squash the commits of the PR when merging (the default).

File: packages/simplescreenrecorder/PKGBUILD (hunk 6 above)
Comment (thisven) [reply packages/simplescreenrecorder/PKGBUILD]: It builds like a charm.

**Pull request merged**